dis_snek/annotations/argument_annotations.py

```diff
@@ -0,0 +1,68 @@
+from typing import Union, TYPE_CHECKING, Any, List, Type, Dict
+
+from dis_snek.const import MISSING
+from dis_snek.utils.misc_utils import get_parameters
+
+from dis_snek.models.context import Context
+from dis_snek.models.scale import Scale
+from dis_snek.client import Snake
+
+if TYPE_CHECKING:
+    from dis_snek.models import Member, User, TYPE_MESSAGEABLE_CHANNEL
+
+
+def define_annotation():
+    """
+    Define a function as an annotation.
+
+    The function **must** type hint its expected arguments. Make sure your annotation runs quickly,
+    as command execution will be delayed until all annotations are completed
+
+    **Supported Types:**
+    `Context`
+    `Scale`
```

Comment:
No `Snake` support?

Reply:
No, remember our conversation in #lib-discussion? Commands dont have a reference to client